soc/xtensa/intel_adsp/ace/sram.c

```diff
@@ -40,10 +40,17 @@ __imr void lp_sram_init(void)
 {
 	uint32_t lpsram_ebb_quantity = ace_lpsram_get_bank_count();
 	volatile uint32_t *l2usbpmptr = (volatile uint32_t *)ACE_L2MM->l2usbpmptr;
+	volatile uint8_t *status = (volatile uint8_t *)l2usbpmptr + 4;
```

Comment:
The 4 smells of a magic number, but we have such magic numbers already in the driver before this PR, so maybe better to change in a follow-up PR.